client/src/lib/repositories/datapass.ts

```diff
@@ -0,0 +1,77 @@
+import type { AuthenticatedUser } from "src/definitions/auth";
+import { SearchParamsValidationError } from "src/definitions/errors";
+import type { Fetch } from "src/definitions/fetch";
+import type { Organization } from "src/definitions/organization";
+import { getApiUrl } from "../fetch";
+
+type DatapassUserPayload = {
+  siret: string;
+  email: string;
+};
+
+type CreateDatapassUser = (opts: {
+  fetch: Fetch;
+  token: string;
+  data: DatapassUserPayload;
+}) => Promise<AuthenticatedUser>;
+
+export const createDatapassUser: CreateDatapassUser = async ({
+  fetch,
+  token,
+  data: { siret, email },
+}) => {
+  const body = JSON.stringify({ organization_siret: siret, email });
+  const url = `${getApiUrl()}/auth/datapass/users`;
```

Comment:
Utiliser un `/` final sinon le back redirigera vers la version avec trailing slash à chaque appel

```suggestion
  const url = `${getApiUrl()}/auth/datapass/users/`;
```